roles/kube_prometheus_stack/files/jsonnet/tests.yml

```
@@ -105,3 +105,18 @@ tests:
       - eval_time: 5m
         alertname: NodeTimeSkewDetected
         exp_alerts: []
+
+  - interval: 1m
+    input_series:
+      - series: 'mysql_up{instance="percona-xtradb-pxc-0", job="pxc"}'
+        values: '1'
+    input_series:
+      - series: 'mysql_up{instance="percona-xtradb-pxc-1", job="pxc"}'
+        values: '1'
+    input_series:
+      - series: 'mysql_up{instance="percona-xtradb-pxc-3", job="pxc"}'
+        values: '0'                
```

```suggestion
    input_series:
      - series: 'mysql_up{instance="percona-xtradb-pxc-0", job="pxc"}'
        values: '1'
      - series: 'mysql_up{instance="percona-xtradb-pxc-1", job="pxc"}'
        values: '1'
      - series: 'mysql_up{instance="percona-xtradb-pxc-3", job="pxc"}'
        values: '0'                
```

You need to make this way otherwise it only processes the last value :)

Also we need to have tests for:

- all instances are up = no alerts
- one replica down = low alert
- multiple replica down = higher alert

etc :)

---

sorry for being so silly, already fix it